src/marketplace/sellers/technical-review-guidelines.md

```diff
@@ -32,6 +32,42 @@ The Marketplace coding standard review uses a custom set of coding sniffs. If th
 
 _See also:_ [Coding Standards]({{ site.baseurl }}/guides/v2.3/coding-standards/bk-coding-standards.html)
 
+### Package Verification: Verify that submitted code is valid Magento extension
```

Comment:
### Package Verification: Verify that submitted code is a valid Magento extension

Reply:
Fixed